mta/mta.sw.yaml

```diff
@@ -150,8 +150,9 @@ states:
           refName: createNotification
           arguments:
             title: "\"MTA analysis report for \" + .application.repository.url"
-            message: '"<a href=\"" + $SECRET.mta_url+ "/hub/applications/" + (.application.id|tostring)+ "/bucket/" + .data.output + "\">MTA Analysis Rerport</a>"'
+            message: '"MTA Analysis Report:" + $SECRET.mta_url+ "/hub/applications/" + (.application.id|tostring)+ "/bucket/" + .data.output '
             origin: "MTA Analysis Workflow"
+            topic: "MTA Analysis"
             targetUsers:
```

Comment:
same: this can be removed IMO

Reply:
Done

Reply:
It is a nice touch:
![Selection_061](https://github.com/parodos-dev/serverless-workflows/assets/316242/80ecdacb-cc7e-4410-aa2b-2e7b52097d87)